--- src/org/jruby/ext/posix/WindowsSecurityAttributes.java.orig	2011-06-02 09:07:29.229447029 -0400
+++ src/org/jruby/ext/posix/WindowsSecurityAttributes.java      2011-06-02 09:06:44.057449528 -0400
@@ -8,14 +8,14 @@ import com.kenai.jaffl.struct.StructUtil
 public class WindowsSecurityAttributes extends HeapStruct {
     public final Unsigned32 length = new Unsigned32();
     public final Pointer securityDescriptor = new Pointer();
-    public final WBOOL inheritHandle = new WBOOL();
+    public final boolean inheritHandle;
 
     public WindowsSecurityAttributes() {
         super();
         
         // This seems like the sensible defaults for this.
         length.set(StructUtil.getSize(this));
-        inheritHandle.set(true);
+        inheritHandle = true;
     }
     
     public long getLength() {
@@ -23,6 +23,6 @@ public class WindowsSecurityAttributes e
     }
 
     public boolean getInheritHandle() {
-        return inheritHandle.get();
+        return inheritHandle;
     }
 }
